webview/src/setup/components/studio/Connect.tsx

```diff
@@ -7,7 +7,7 @@ import { Button } from '../../../shared/components/button/Button'
 export const Connect: React.FC = () => {
   return (
     <EmptyState isFullScreen={false}>
-      <div data-testid="setup-studio-content">
```

Comment:
Pretty unrelated to this pr since I'm no longer updating this section, but we're not using the test id anywhere.